pymeasure/instruments/lecroy/lecroyT3DSO1204.py

```diff
@@ -1030,6 +1037,20 @@ def download_data(self, source, requested_points=0, sparsing=1):
 
     _trigger_select = Instrument.control(
         "TRSE?", _trigger_select_normal_command,
+        """ Refer to the self.trigger_select documentation. """,
+        get_process=_trigger_select_get_process,
+        validator=_trigger_select_validator,
+        values=_trigger_select_values,
```

Comment:
Does this work? Because you cannot read the "_values" of the dynamic property "_trigger_select", I think. Anyway, `_trigger_select_values` is the `values` attribute of `_trigger_select`, such that you make something like `values = values`.

Reply:
Yes, it works. If I read the `Instrument.control` method source code correctly there is no `values` attribute. `values` is just a local variable of the getter and setter methods of the property.

Reply:
That's "dynamic property magic". If you execute `_trigger_select_values = (1,2,3)` afterwards, the values for the `_trigger_select` property will be changed to that tuple.

I think it works, because you define `_trigger_select_values` and validator etc. **before** you initialize the class (where these values would be protected in order to make the magic work).

Just that you know, once the class is initialized, you cannot read `_trigger_select_values` anymore!

@bilderbuchi should `_trigger_select_values` be renamed to not conflict?

Reply:
@bmoneke @LastStarDust sorry, did not see this earlier. 
Yeah, it would be safer to rename, just to avoid that the "magic" accidentally escapes.
`_trigger_select_vals`, maybe?

Reply:
I renamed `_trigger_select_values` into `_trigger_select_vals`